build-system/tasks/coverage-map/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs').promises;
+const log = require('fancy-log');
+const {dist} = require('../dist');
+const {installPackages} = require('../../common/utils');
+const {startServer, stopServer} = require('../serve');
+
+let puppeteer;
+let explore;
+
+const coverageJsonName = argv.json || 'out.json';
```

Comment:
nit: I know `out.*` is a pretty conventional example name for an output file, but `coverage.*` or something similar might be more intuitive/useful in context

Reply:
I'll fix this.